app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,78 @@
+class FriendshipsController < ApplicationController
+  before_action :set_friendship, only: [:show, :edit, :update, :destroy]
+
+  # GET /friendships
+  # GET /friendships.json
+  def index
+    @friendships = Friendship.all
+  end
+
+  # GET /friendships/1
+  # GET /friendships/1.json
+  def show
+  end
+
+  # GET /friendships/new
+  def new
+    @friendship = Friendship.new
+  end
+
+  # GET /friendships/1/edit
+  def edit
+  end
+
+  # POST /friendships
+  # POST /friendships.json
+  def create
+    @friendship = Friendship.new(friendship_params)
```

Comment:
@sdsunjay you can do `current_user.build_friendship` I believe so you don't have to merge params.

Reply:
So instead of line 27, I could just do `@friendship = current_user.build_friendship`?

Reply:
I actually do want admins to be able to create friendships for other users

Reply:
Yes that’s correct